### DISCUSSION TITLE
ARM-Version of Cider for Asahi Linux

### DISCUSSION THREAD

**Discussion by JustforBugRep** [ciderapp/Cider#1791] [Feature Request]
The developers of Asahi Linux (a project to bring Linux to Apple Silicon Macs) have managed to get widevine support work on aarch64: https://github.com/AsahiLinux/widevine-installer/blob/main/README.md

A version of Cider would be very nice for people who wants to use Apple Music on Mac but without using MacOs.

**Comment by alexhaydock** [comment]
I'm definitely also interested. But just in general it would be nice to have an aarch64 Linux build if possible.
